R/derive_var_atoxgr.R

```diff
@@ -15,14 +15,17 @@
 #'
 #' @param meta_criteria Metadata data set holding the criteria (normally a case statement)
 #'
-#' Permitted Values: atoxgr_criteria_ctcv4, atoxgr_criteria_ctcv5
+#' Permitted Values: atoxgr_criteria_ctcv4, atoxgr_criteria_ctcv5, atoxgr_criteria_daids
 #'
 #'   {admiral} metadata data set `atoxgr_criteria_ctcv4` implements
 #'   [Common Terminology Criteria for Adverse Events (CTCAE)
 #'    v4.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm)
 #'   {admiral} metadata data set `atoxgr_criteria_ctcv5` implements
 #'   [Common Terminology Criteria for Adverse Events (CTCAE)
 #'    v5.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm)
+#'   {admiral} metadata data set `atoxgr_criteria_daids` implements
+#'   [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric
+#'    Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

Comment:
```suggestion
#' - `atoxgr_criteria_ctcv4` implements [Common Terminology Criteria for Adverse #' Events (CTCAE) v4.0]
    (https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm)
#' - `atoxgr_criteria_ctcv5` implements [Common Terminology Criteria for Adverse #' Events (CTCAE) v5.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm)
#' `atoxgr_criteria_daids` implements [Division of AIDS (DAIDS) Table for Grading #' #' the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

Little more succinct?  I did this in GitHub - not sure if it will render correctly?

Reply:
Updated